app/views/purchases/_purchase.html.erb

```diff
@@ -0,0 +1,12 @@
+<tr>
+  <td ><%= purchase.name %></td>
+  <td>$<%= purchase.amount %></td>
+  <td > <%= purchase.category.name %></td>
+  <td >
```

Comment:
- [Optional] I think you shouldn't have displayed this table if no purchase has been made.

![Screenshot from 2022-10-18 18-19-42](https://user-images.githubusercontent.com/63087491/196500779-1733893c-737c-4ebd-a4ee-a154b25fe93b.png)